pkg/controllers/user/networkpolicy/policy.go

```diff
@@ -0,0 +1,346 @@
+package networkpolicy
+
+import (
+	"fmt"
+	"reflect"
+
+	typescorev1 "github.com/rancher/types/apis/core/v1"
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/rancher/types/config"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	networkingv1 "k8s.io/api/networking/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"strings"
+)
+
+const (
+	projectIDAnnotation   = "field.cattle.io/projectId"
+	projectIDNoColonLabel = "management.cattle.io/projectIdNoColon"
```

Comment:
this is not a good name.  What is this used for?

Reply:
I am adding a label to the network policy using the project id + namespace id but without colon which will be used later to do a lookup using the label.

Reply:
just change to `management.cattle.io/projectName`